src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "layouts/default.njk" %}
+{% block main_content %}
+
+  <main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <form class="form" action="address-correspondence-confirm" method="post">
+
+          <span class="govuk-caption-l">{{firstName}} {{lastName}}</span>
+
+          <h1 class="govuk-heading-l">Confirm the correspondence address</h1>
+          <ul class="govuk-list">
+              <li>{{correspondenceAddress.propertyDetails}}</li>
+              <li>{{correspondenceAddress.line1}}</li>
+              <li>{{correspondenceAddress.line2}}</li>
+              <li>{{correspondenceAddress.town}}</li>
+              <li>{{correspondenceAddress.county}}</li>
+              <li>{{correspondenceAddress.country}}</li>
+              <li>{{correspondenceAddress.postcode}}</li>
+          </ul>
+
+          <p class="govuk-body">
+            <a href="correspondance-address-manual" class="govuk-link govuk-link--no-visited-state">Edit address</a>
```

Comment:
can this href be pulled from the pageURL ?

Reply:
PageURLs is not about localisation. what will happens if you add this in pageURLs?